plugins/builds/update.js

```diff
@@ -68,62 +68,63 @@ module.exports = () => ({
                     build.status = desiredStatus;
 
                     // Only trigger next build on success
-                    return build.job.then(job => job.pipeline.then((pipeline) => {
-                        request.server.emit('build_status', {
-                            settings: job.permutations[0].settings,
-                            status: build.status,
-                            pipelineName: pipeline.scmRepo.name,
-                            jobName: job.name,
-                            buildId: build.id,
-                            buildLink: `${buildFactory.uiUri}/builds/${id}`
-                        });
-
-                        // Guard against triggering non-successful builds
-                        if (desiredStatus !== 'SUCCESS') {
-                            return null;
-                        }
-
-                        const workflow = pipeline.workflow;
+                    return build.update()
```

Comment:
In my local test environment, parent build status is still RUNNING at the beginning of `sd-setup-builder`.
How is your local environment?
I think that It looks need waiting status change.